turn_alerts/tests/test_views.py

```diff
@@ -0,0 +1,384 @@
+import base64
+import hmac
+import json
+from hashlib import sha256
+from unittest.mock import Mock, patch
+
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.renderers import JSONRenderer
+from rest_framework.test import APIRequestFactory, APITestCase
+
+from sidekick.models import Organization
+from sidekick.tests.utils import create_org
+from turn_alerts.models import TurnActions
+from turn_alerts.views import TurnAlertsLayerView
+
+from .utils import create_turnalerts
+
+
+class TestAlertsViewAbstract(APITestCase):
+    def setUp(self):
+        user = get_user_model().objects.create_user("test")
+        self.client.force_authenticate(user)
+
+        self.org = create_org()
+        self.turnalerts_account = create_turnalerts(org=self.org)
```

Comment:
This isn't being used

Reply:
create_org() creates the org object that is being used here:

_url = reverse("turn-messages", kwargs={'org_id' : self.org.id})_